examples/access-control-migration/README.md

```diff
@@ -1,6 +1,306 @@
 # @loopback/example-access-control-migration
 
-This is the basic tutorial for getting started with Loopback 4!
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
+
+This example is a donation system. Users can view or make donations to projects
+based on their roles. A project is owned by a user, and a user can create a team
+to involve other users as team members. The system has an administration.
+
+Here is the diagram that describes the models:
+
+![models](./imgs/scenario-model.png)
+
+And the Project endpoints with their allowed roles:
+
+- /Projects/list-projects
+  - everyone
+- /Projects/view-all
+  - admin
+- /Projects/{id}/show-balance
+  - owner, teamMember
+- /Projects/{id}/donate
+  - admin, owner, teamMember
+- /Projects/{id}/withdraw
+  - owner
+
+The diagram for models also include the pre-created data in this application.
```

Comment:
Suggestion:
This application includes some pre-created data. They are shown in the above diagram. As you can see, u1 owns project1...

Feel free to ignore

Reply:
more fluent 👍